### ISSUE TITLE
svn to git migration of Mathcad code

### ISSUE THREAD

**Issue by monroews** [AguaClara-Reach/aguaclara#89]
I had assumed we wouldn't need to keep the Mathcad code available with the transition to python, but there is a current need to be able to update designs and thus we need to keep Mathcad code available. We also need the ability to select particular version numbers for revision and thus we need to port the entire mathcad code base to github. 

[Directions for doing this are here.](https://confluence.cornell.edu/display/coecis/Github+Enterprise+Repository%2C+COE+CIS+TECH+IT+Support)

This migration will allow us to access old versions of the code.

We have until September 30, so the clock is ticking.

I wasn't sure who to assign this to. Perhaps Oliver can take the lead?

@ethan92429 @juanchon998  @oliver-leung

**Comment by eak24** [comment]
@oliver-leung and @juanchon998 already have a [git repo of MathCAD](https://github.com/AguaClara/mathcad_Final_Designs)

**Comment by eak24** [comment]
Am I right in assuming there should be a repo for each of these projects?:

![screen shot 2018-08-28 at 10 28 34](https://user-images.githubusercontent.com/8585379/44729661-4b0ce300-aaad-11e8-9e6c-ab22cabad45c.png)

**Comment by eak24** [comment]
Thanks for reopening Juan, Monroe and I just discussed that the repo that had been created didn't have past versions. I can take this on right now.

**Comment by juanchon998** [comment]
Thank you! I was typing out a comment but got distracted as everyone in the office wanted to try some mate. I would really appreciate a git-based way to access previous versions of the design code! In particular I need to access version 6807 for the Las Moras design.

**Comment by monroews** [comment]
I think the only repo that is needed is the Automated Design. I think the others are all many years old and no longer needed.

**Comment by eak24** [comment]
My computer is on revision 2107... This makes me realize just how long we've been iterating on our MathCAD code. Hats off to Monroe for keeping this system going for 7k+ iterations... NumPy is at 18k commits, so we're within a factor of 3!

**Comment by eak24** [comment]
This is a bit more challenging than anticipated. GitHub doesn't like large files, yet over the years in svn we've had a ton of large files. So, GitHub is throwing an error when I try to upload the repo. I downloaded git lfs (large file system) to take care of this issue, but the error kept rearing its head. [I've turned to stackoverflow for suggestions.](https://stackoverflow.com/questions/52063592/migrate-an-svn-repo-with-large-files-to-a-new-git-repo)
